src/docs/product/sentry-basics/guides/alert-notifications/index.mdx

```diff
@@ -6,38 +6,39 @@ redirect_from:
 description: "Learn more about how to both understand and manage the alerts and notifications that give real-time visibility into errors happening in your code."
 ---
 
-Sentry’s alert notifications give real-time visibility into errors happening in your code and the impact they are having on your users regardless of where your code is deployed. They explain why you’re being notified and uncover meaningful and customizable information about the error and its context. In addition, Sentry will also notify your project team members about various workflow activities.
+Sentry’s alert notifications give real-time visibility into errors happening in your code and the impact on your users regardless of where your code is deployed. Alerts explain why you’re being notified and uncover meaningful and customizable information about the error and its context. In addition, Sentry will notify your project team members about various workflow activities.
 
-## **Alert Notifications**
+## Alerts
 
-When you create a new project in Sentry, a **default alert rule** populates within the project's Alerts view. The default alert notifies **all** project team members any time an error is **first** encountered (for example, when a new **issue** is created) in this project. This notification is sent via email and any other _Legacy Integration_ services you might have enabled.
+When you create a new project in Sentry, you can choose from a few default alerts or create one later. To create an alert, navigate to **Alerts > New Alert Rule** and choose between a [Metric Alert](/product/sentry-basics/guides/alert-notifications/metric-alerts/) or an [Issue Alert](/product/sentry-basics/guides/alert-notifications/issue-alerts/).
 
-![Default Alert](alerts_notifications_01.png)
+![Choice between Metric Alert or Issue Alert](create-new-alert-rule.png)
 
-Getting notified **once** when an error is first seen is generally useful for catching new errors in real-time, but will not notify you about subsequent instances of the same errors. This means that with only the default alert rule in place, you might be losing visibility into critical issues building up in your code.
+Metric alerts are usually a higher signal than issue alerts because you know upfront what you're monitoring. In contrast, issue alerts power a triage workflow — you cast a wide net for potential problems and review them as they happen.
```

Comment:
I'm not sure what we mean by "higher signal" here - it comes across as jargon - can we simplify?